genie_annotations.json

```diff
@@ -0,0 +1,24 @@
+{
+  "tissueSource": [
+    "Blood",
+    "Solid Tumor"
+  ],
+  "center": [
+    "NKI",
+    "DFCI",
+    "GRCC",
+    "JHU",
+    "MSK",
+    "UHN",
+    "VICC",
+    "MDA"
+  ],
+  "platform": [
```

Comment:
True, but enabling discovery of data on the same platform across projects is important. We should draw from a standardized source of platform names; `HiSeq2000` should be, at a minimum, `IlluminaHiSeq2000`, for example (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL11154)